test/blackbox/common/BlackboxTestsPubSubHistory.cpp

```diff
@@ -664,6 +664,69 @@ TEST_P(PubSubHistory, ReliableTransientLocalKeepLast1Data300Kb)
     ASSERT_EQ(reader.get_last_sequence_received().low, 10u);
 }
 
+// Test created to check bug #8945
```

Comment:
Should we really use 1600 samples for this? Is it reproducible with a lower number, like 16 ?